plugin/environment.go

```diff
@@ -300,10 +301,14 @@ func (env *Environment) Shutdown() {
 		ap := value.(activePlugin)
 
 		if ap.supervisor != nil {
-			if err := ap.supervisor.Hooks().OnDeactivate(); err != nil {
-				env.logger.Error("Plugin OnDeactivate() error", mlog.String("plugin_id", ap.BundleInfo.Manifest.Id), mlog.Err(err))
+			//if err := ap.supervisor.Hooks().OnDeactivate(); err != nil {
+			//	env.logger.Error("Plugin OnDeactivate() error", mlog.String("plugin_id", ap.BundleInfo.Manifest.Id), mlog.Err(err))
+			//}
+			//ap.supervisor.Shutdown()
```

Comment:
Why is ap.supervisor.Shutdown() commented out?

Reply:
@levb it was left it to show that I would prefer it there...  However if either `			//ap.supervisor.Shutdown()`, or `ap.supervisor.Hooks().OnDeactivate()` is left in, the `p.Signal` call doesn't seem to fire or effectively kill the plugin process.